core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.directive.html

```diff
@@ -54,7 +54,7 @@
             Assigned
           </span>
           <span ng-if="!skill.topicName" class="skill-topic-unassigned protractor-test-assign-skill-to-topic-button" ng-click="$ctrl.assignSkillToTopic(skill.id)">
-            Unassigned
+            UNASSIGNED
```

Comment:
Why this change? (Why this isn't similar to `Assigned` on line 54?)

Reply:
This was a suggestion from @mschanteltc. Basically, this is an actionable text/button and hence it's capitable, while Assigned is just plain information. See the issue description [here](https://github.com/oppia/oppia/issues/10126), there's a bullet point for that.